components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -24,7 +24,7 @@ use rand::Rng;
 
 use export_docker::Result;
 
-use topology::Topology;
+use habitat_sup::manager::service::Topology;
```

Comment:
OK for now, but I wonder if longer-term we might want to pull this type up to a common / core crate instead of depending directly on the supervisor crate.